src/android/CameraLauncher.java

```diff
@@ -632,73 +645,79 @@ else if (destType == FILE_URI || destType == NATIVE_URI) {
      */
     public void onActivityResult(int requestCode, int resultCode, Intent intent) {
 
-        // Get src and dest types from request code for a Camera Activity
-        int srcType = (requestCode / 16) - 1;
-        int destType = (requestCode % 16) - 1;
-
-        // If Camera Crop
-        if (requestCode >= CROP_CAMERA) {
-            if (resultCode == Activity.RESULT_OK) {
+        if (this.imageUri == null && this.getPictureFromGallery == false) {
```

Comment:
Would it be more appropriate to check if `callbackContext == null`? An app can be evicted even when getting a picture from the gallery.